addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -298,7 +315,22 @@
                 QWeb.render('slide.slide.quiz.validation', {'widget': this})
             );
         },
-
+        /*
+        * @private
+        * Show additional resource info box
+        */
+        _renderAdditionalResourceInfo: function() {
+            var $resourceInfo = $('div.o_wslides_js_lesson_quiz_resourceinfo');
+            $resourceInfo.removeClass('d-none');
+        },
+        /*
+        * @private
+        * Show additional resource info box
+        */
+        _hideAdditionalResourceInfo: function() {
+            var $resourceInfo = $('div.o_wslides_js_lesson_quiz_resourceinfo');
+            $resourceInfo.addClass('d-none');
+        },
```

Comment:
I'd apply a few things here:
* make a toggle function with `on` flag, as there is only one difference between the methods
* use `const` for "variables" that won't change reference, such as `resourceInfoDiv`
* avoid using jQuery as we 'll be migrating out of it, let's not add more work for later